isobus/src/isobus_virtual_terminal_client.cpp

```diff
@@ -2286,6 +2313,50 @@ namespace isobus
 		                                                      CANIdentifier::PriorityLowest7);
 	}
 
+	bool VirtualTerminalClient::send_aux_n_preferred_assignment()
+	{
+		// TODO load preferred assignment from saved configuration
+		// TODO only send command if there is an Auxiliary Function Type 2 object in the object pool
```

Comment:
Nit: Can you make the TODOs here and later in the file `/// @todo` so doxygen picks it up?